.github/workflows/update-branding-assets.yml

```diff
@@ -0,0 +1,227 @@
+              mkdir -p "application/public/brand/$(dirname "$logo_path")"
+              cp "/tmp/branding-assets/$logo_path" "application/public/brand/$logo_path"
+              echo "✓ Copied logo: $logo_path"
+            else
+              echo "::warning::Logo file not found: $logo_path"
+            fi
+          done
+          
+          # Copy mascot (if present)
+          MASCOT_FILE=$(echo "$MANIFEST" | jq -r '.mascot.file // empty')
+          if [ -n "$MASCOT_FILE" ] && [ -f "/tmp/branding-assets/$MASCOT_FILE" ]; then
+            mkdir -p "application/public/brand/$(dirname "$MASCOT_FILE")"
+            cp "/tmp/branding-assets/$MASCOT_FILE" "application/public/brand/$MASCOT_FILE"
+            echo "✓ Copied mascot: $MASCOT_FILE"
+          fi
+          
+          # Save manifest for reference
+          echo "$MANIFEST" > application/src/data/brand/manifest.json
+          echo "✓ Saved manifest"
+
+      - name: Create logo pack
+        uses: ./.github/actions/create-logo-pack
+
+      - name: Setup Node.js
+        uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af # v4.1.0
+        with:
+          node-version: '20'
+          cache: 'npm'
+          cache-dependency-path: 'application/package-lock.json'
+
+      - name: Install dependencies
+        run: |
+          cd application
+          npm ci
+
+      - name: Generate brand guidelines PDF
+        uses: ./.github/actions/generate-brand-pdf
+        with:
+          page-url: 'http://localhost:4321/charte-graphique'
+          output-file: 'application/public/brand/downloads/hoverkraft-brand-guidelines.pdf'
+          working-directory: 'application'
+
+      - name: Validate updated assets
+        run: |
+          echo "Validating branding assets..."
+          
+          MISSING_ASSETS=()
+          
+          # Check for generated colors
+          if [ ! -f "application/src/data/brand/generated-colors.ts" ]; then
+            MISSING_ASSETS+=("generated-colors.ts")
+          fi
+          
+          # Check for at least one logo
+          if [ ! -d "application/public/brand/logo" ] || [ -z "$(find application/public/brand/logo -type f)" ]; then
+            MISSING_ASSETS+=("logos")
+          fi
+          
+          # Check for logo pack
+          if [ ! -f "application/public/brand/downloads/hoverkraft-logos.zip" ]; then
+            MISSING_ASSETS+=("hoverkraft-logos.zip")
+          fi
+          
+          # Check for generated PDF
+          if [ ! -f "application/public/brand/downloads/hoverkraft-brand-guidelines.pdf" ]; then
+            MISSING_ASSETS+=("hoverkraft-brand-guidelines.pdf")
+          fi
+          
+          if [ ${#MISSING_ASSETS[@]} -gt 0 ]; then
+            echo "::error::Missing required branding assets: ${MISSING_ASSETS[*]}"
+            exit 1
+          fi
+          
+          echo "✓ All required branding assets are present"
+
+      - name: Commit updated assets
```

Comment:
use hoverkraft-tech/ci-github-common/actions/create-and-merge-pull-request action instead